### DISCUSSION TITLE
guide/015/

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/scriptdev.ru#126] [Announcements]
# guide/015/

Справочник Typescript с примерами кода

https://scriptdev.ru/guide/015/

**Comment by akambetov** [comment]
https://scriptdev.ru/guide/015/#_4
Перечисление в качестве типа данных
Цитата: 
"Функцию, тип параметра которой является смешанным перечислением, благополучно получится вызвать как с константой перечисления в качестве аргумента, так и с любым числом."

С пятой версии TS компилятор не пропустит вызов такой функции (параметр с типом смешанный/числовой enum) с аргументом number
